meta(changelog): Update changelog for 10.9.0 (#17518)
getsentry/sentry-javascript: CHANGELOG.md

- "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott

## 10.9.0

### Important Changes

- **feat(node): Update `httpIntegration` handling of incoming requests ([#17371](https://github.com/getsentry/sentry-javascript/pull/17371))**

This version updates the handling of the Node SDK of incoming requests. Instead of relying on @opentelemetry/instrumentation-http, we now handle incoming request instrumentation internally, ensuring that we can optimize performance as much as possible and avoid interop problems.

> This change should not affect users, unless they are relying on very in-depth implementation details. Importantly, this also drops the `_experimentalConfig` option of the integration - this will no longer do anything.

Reviewer: ```suggestion
This change should not you, unless you're relying on very in-depth implementation details. Importantly, this also drops the `_experimentalConfig` option of the integration - this will no longer do anything.
```

Author: Updated, thanks!
